data/ga/legislature/Marcus-Wiedower-93826349-e5d6-47fc-a7c6-548257629272.yml

```diff
@@ -0,0 +1,38 @@
+id: ocd-person/93826349-e5d6-47fc-a7c6-548257629272
+name: Marcus Wiedower
+given_name: Marcus
+family_name: Wiedower
+email: marcus.wiedower@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/wiedower-marcus-4956.jpg
+party:
+- name: Republican
+roles:
+- type: lower
+  jurisdiction: ocd-jurisdiction/country:us/state:ga/government
+  district: '121'
```

Comment:
@sroomf this person was a representative for district 119 from Jan 2019 until Jan 2023.